app/models/vm_or_template/operations.rb

```diff
@@ -112,11 +112,10 @@ def validate_vm_control
             end
       unsupported_reason_add(:control, msg) if msg
     end
-  end
 
-  def validate_terminate
-    return {:available => false, :message => 'The VM is terminated'} if self.terminated?
-    {:available => true, :message => nil}
+    supports :terminate do
```

Comment:
Ok, looking at the original commit that introduced this 378c5d6bbc88976b632f1bd4cee4805b13900db2 and the PR https://github.com/ManageIQ/manageiq/pull/6475
It looks like the `terminate` operation is actuallty [vm_destroy](https://github.com/ManageIQ/manageiq/blob/009be628e06c689d8f2499cff38ef93051dc1d9e/app/controllers/application_controller/ci_processing.rb#L1848) - which is only supported if a cloudmanager implements `vm_destroy`